### DISCUSSION TITLE
Backup / restore watch state?

### DISCUSSION THREAD

**Discussion by devanteweary** [luigi311/JellyPlex-Watched#121] [Q&A]
I have it syncing between Plex and Jellyfin and it mostly works OK (some errors when trying to sync).

However, how do I use this to make a backup of my watchstates and then restore later if needed?

thank you!

**Comment by luigi311** [comment]
As in store a backup of all your watched states so you can later restore that backup? That isnt something ive implemented though i guess i already have most of it setup to allow me to do so. I dont know how i would structure it though but go ahead and open up a feature request for it as an issue.
